src/api/spec/models/staging/workflow_spec.rb

```diff
@@ -23,13 +24,23 @@
       it { expect(subject).to be_empty }
     end
 
-    context 'with requests but not in staging projects' do
+    context 'with requests without reviews by the staging managers group' do
       before { bs_request }
 
+      it { expect(subject).to be_empty }
+    end
+
+    context 'with requests with reviews by the staging managers group' do
+      let(:review) { create(:review, by_group: group.title) }
+
+      before do
+        bs_request.reviews << review
```

Comment:
Why not `let!(:review) { create(:review, by_group: group.title, bs_request: bs_request) }` ?